<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtpArbClass</class>
 <widget class="QMainWindow" name="CtpArbClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CtpArb</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_work" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="widget_contract1" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 225, 225);</string>
              </property>
              <widget class="QWidget" name="">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>0</y>
                 <width>291</width>
                 <height>25</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="styleSheet">
                   <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
                  </property>
                  <property name="text">
                   <string>合约A：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_ins1">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>合约编码</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_ins1">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_27">
                  <property name="text">
                   <string>手数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_vol1">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>80</x>
                 <y>40</y>
                 <width>239</width>
                 <height>20</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QComboBox" name="comboBox_ptype1">
                  <item>
                   <property name="text">
                    <string>对盘价</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>排队价</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>最新价</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>市价</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_28">
                  <property name="text">
                   <string>+</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_slip1"/>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>点</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_chase1">
                  <property name="text">
                   <string>追单</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_chase1_point"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_contract2" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(200, 237, 206);</string>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>81</x>
                 <y>45</y>
                 <width>241</width>
                 <height>20</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QComboBox" name="comboBox_ptype2">
                  <item>
                   <property name="text">
                    <string>对盘价</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>排队价</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>最新价</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>市价</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>+</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_slip2"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>点</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_chase2">
                  <property name="text">
                   <string>追单</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_chase2_point"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>10</y>
                 <width>291</width>
                 <height>25</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="styleSheet">
                   <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
                  </property>
                  <property name="text">
                   <string>合约B：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_ins2">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>合约编码</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_ins2">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>手数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_vol2">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget_area2" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 252, 247);</string>
            </property>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>110</y>
               <width>131</width>
               <height>23</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 185, 39);</string>
             </property>
             <property name="text">
              <string>提交订单</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget_7">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>9</y>
               <width>251</width>
               <height>20</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>开平：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_offset">
                <item>
                 <property name="text">
                  <string>开仓</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>平仓</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>类型：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_send_order_type">
                <item>
                 <property name="text">
                  <string>全部同时</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>主动腿-A</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>主动腿-B</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>先腿-A</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>先腿-B</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_8">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>298</width>
               <height>22</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>公式一：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_formula1">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>A-B</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_cmp1">
                <item>
                 <property name="text">
                  <string>&gt;=</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>&lt;=</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_formula1_cmp">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_9">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>375</width>
               <height>22</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>公式二：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_formula2">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>A-B</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_cmp2">
                <item>
                 <property name="text">
                  <string>&gt;=</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>&lt;=</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_formula2_cmp">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_loop">
                <property name="text">
                 <string>循环套利</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="pushButton_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>110</y>
               <width>81</width>
               <height>23</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 185, 39);</string>
             </property>
             <property name="text">
              <string>新增</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QTreeView" name="treeView">
       <property name="styleSheet">
        <string notr="true">border:1px solid rgb(85, 170, 255);
background-color: rgb(255, 247, 239);</string>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CtpArb.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mainToolBar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>CtpArbClass</receiver>
   <slot>ShowLoginForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>6</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>CtpArbClass</receiver>
   <slot>ChooseInstrument(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ins1</sender>
   <signal>clicked()</signal>
   <receiver>CtpArbClass</receiver>
   <slot>ShowInstrument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ins2</sender>
   <signal>clicked()</signal>
   <receiver>CtpArbClass</receiver>
   <slot>ShowInstrument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ShowLoginForm()</slot>
  <slot>ShowInstrument()</slot>
  <slot>ChooseInstrument(QModelIndex)</slot>
 </slots>
</ui>
